test/TEST-99-GETARG/test.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
```

Comment:
Minor nitpick add proper header for new files ( even if it is an test ) it's missing
```
# This file is part of dracut.
# SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
will add